inspectors/types/noteheadtypes.h

```diff
@@ -69,11 +69,20 @@ class NoteHeadTypes
         SCHEME_SHAPE_NOTE_7_WALKER
     };
 
+    enum class ParenthesesFlags : signed char {
```

Comment:
No need to do `signed char`.

Reply:
however, for char it is better to be explicit, as it is implementation defined whether a plain char is signed or unsigned

Reply:
According to @vpereverzev, it's exactly the recasting to `int` that makes `signed char` meaningless. Is it not that simple?

Reply:
But just `char` won't make it any better

Reply:
What if no `char` at all?

Reply:
Then it'd "waste" 3 more bytes, defaulting to int.

Reply:
or leave it blank and hope the compiler will optimize?

Reply:
No compiler in the world ever would (be allowed to). Not specifying a size means `int`.